worker-message-prioritization-rerouting/src/main/java/com/github/workerframework/workermessageprioritization/rerouting/MessageRouterSingleton.java

```diff
@@ -101,6 +103,22 @@ public static String route(final String originalQueueName, final String tenantId
         }
     }
 
+    public static void checkHealth(final HealthMonitor healthMonitor) {
+        if (CAF_WMP_ENABLED) {
+            try {
+                final RabbitManagementApi<HealthCheckApi> healthCheckApi = new RabbitManagementApi<>(
+                        HealthCheckApi.class, CAF_RABBITMQ_MGMT_URL, CAF_RABBITMQ_MGMT_USERNAME, CAF_RABBITMQ_MGMT_PASSWORD);
+
+                healthCheckApi.getApi().checkHealth();
+            } catch (final Throwable t) {
+                final String errorMessage = String.format(
+                        "RabbitMQ Management API healthcheck failed. Exception message: %s", t.getMessage());
+                healthMonitor.reportUnhealthy(errorMessage);
```

Comment:
Chatted this through with Dermot and because the we're allowing messages to continue to flow through the worker when there is a problem with the ManagemenentAPI we should not fail the healthcheck for that reason.
Comment out healthMonitor.reportUnhealthy(errorMessage); but continue to log the error.

Reply:
Ok, if we're not reporting unhealthy, is there any point in logging the error though, as an error will be logged in the next call to `route` if there is a problem with the API, either here:

https://github.com/WorkerFramework/worker-message-prioritization/blob/main/worker-message-prioritization-rerouting/src/main/java/com/github/workerframework/workermessageprioritization/rerouting/MessageRouter.java#L123

or here:

https://github.com/WorkerFramework/worker-message-prioritization/blob/main/worker-message-prioritization-rerouting/src/main/java/com/github/workerframework/workermessageprioritization/rerouting/MessageRouter.java#L139

Reply:
Just back from standup, think I will go with your suggestion (logging an error). Dermot mentioned it might be good to keep the healthcheck stuff in place as the PRs are already done, in case we do want to fail the healthcheck in future.